devel/buf/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; -*-
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        bufbuild buf v1.50.1
+github.tarball_from archive
+
+categories          devel
+license             Apache-2.0
+maintainers         @mikaoj openmaintainer
+description         Buf is a tool for working with Protocol Buffers.
+long_description    The Buf CLI is a helpful tool for managing Protobuf schemas. \
+                    It offers various features, including code generation, breaking change detection, linting, and formatting, \
+                    to assist with Protobuf development and maintenance. \
+                    It works with your choice of plugins and languages and gives you access to a vast library of certified plugins in the Buf Schema Registry.
+homepage            https://buf.build/
+
+checksums           rmd160  be72fcd551d797aeb73f26060569c282e383e785 \
+                    sha256  2dc0e7eae6a9cc206de4421162e0f5895b9488a1614b8bf30eebd5588cd08df5 \
+                    size    1590934
+
+depends_build       port:go
```

Comment:
should/could this port use the "golang" PortGroup?

Reply:
Sure, fixed

Reply:
Now that you're using the `golang` portgroup, you no longer need this `depends_build       port:go` line.  The portgroup specifies that by default.

Also, you may need to add `go.offline_build no` directly below the `go.setup` line, because by default, the `golang` portgroup wants to see dependencies specified using a `go.vendors` block.

Reply:
Removed depends_build.
Had already added go.offline_build when I switched to golang portgroup 😃